### PR TITLE
maintain: clean up architecture...and the rest

### PR DESCRIPTION
**Pull request by technovangelist**
Signed-off-by: Matt Williams <m@technovangelist.com>

## Summary

The architecture page had issues, so fixed them up. And then looked at active vs passive voice, better grammar throughout, and so much more.

## Checklist

- [x] Wrote appropriate unit tests
- [x] Considered security implications of the change
- [x] Updated associated docs where necessary
- [x] Updated associated configuration where necessary
- [x] Change is backwards compatible if it needs to be (user can upgrade without manual steps?)
- [x] Nothing sensitive logged
- [x] Considered data migrations for smooth upgrades

## Related Issues

Resolves #

### CODE REVIEW

File: docs/manage/connectors/kubernetes.md [hunk 1]
```diff
@@ -42,17 +42,48 @@ helm upgrade --install infra-connector infrahq/infra \
 
 {% callout type="info" %}
 
-Note: it may take a few minutes for the LoadBalancer to be provisioned.
+Note: it may take a few minutes for the Load Balancer to be provisioned.
```
Comment (mxyng) [inline docs/manage/connectors/kubernetes.md:45]: This shouldn't be changed since it's the Kubernetes resource name

File: docs/reference/cli.md [hunk 2]
```diff
@@ -2,812 +2,835 @@
 title: CLI Reference
```
Comment (mxyng) [inline docs/reference/cli.md:2]: This is generated so it shouldn't be manually changed

Comment (technovangelist) [reply docs/reference/cli.md:2]: Oh, how is it generated? I can update that instead

Comment (mxyng) [reply docs/reference/cli.md:2]: It's generated by `internal/docgen`

**Review by mchiang0610 — APPROVED** [review]
saw @mxyng's comment above per CLI docs need to be generated. Let's get that in. Otherwise this is a step forward, and we can make more iterations as we go.

**Pull request merged**